esp/xslt/esdl2ecl.xslt

```diff
@@ -17,26 +17,57 @@
 ##############################################################################
 -->
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-	<xsl:output method="text" version="1.0" encoding="UTF-8" indent="yes"/>
-	<xsl:param name="sourceFileName" select="'UNKNOWN'"/>
-	<xsl:variable name="docname" select="/expesdl/esxdl/@name"/>
-	<xsl:template match="/">
-		<xsl:apply-templates select="expesdl"/>
-	</xsl:template>
-	<xsl:template name="doNotChangeManuallyComment">
-		<xsl:text>/*** Not to be hand edited (changes will be lost on re-generation) ***/
+    <xsl:output method="text" version="1.0" encoding="UTF-8" indent="yes"/>
+    <xsl:param name="sourceFileName" select="'UNKNOWN'"/>
+    <xsl:param name="importsList" select="''"/>
+    <xsl:variable name="docname" select="/expesdl/esxdl/@name"/>
+    <xsl:template match="/">
+        <xsl:apply-templates select="expesdl"/>
+    </xsl:template>
+    <xsl:template match="text()" name="outputImports">
+    <xsl:param name="import" select="."/>
+        <xsl:if test="string-length($import) > 0">
+            <xsl:text>import $.</xsl:text><xsl:value-of select="$import"/><xsl:text>;
```

Comment:
Actually since the import mechanism would be ECL specific maybe we should pass in the whole import, including the "$."?  Seems like that would give more flexibility.